src/plopp/backends/plotly/canvas.py

```diff
@@ -240,6 +240,17 @@ def pan(self):
         """
         self.fig.update_layout(dragmode='pan')
 
+    def panzoom(self, value: Literal['pan', 'zoom', None]):
+        """
+        Activate or deactivate the pan or zoom tool, depending on the input value.
+        """
+        if value == 'zoom':
+            self.zoom()
+        elif value == 'pan':
+            self.pan()
+        elif value is None:
+            self.reset_mode()
```

Comment:
This is duplicated now. Can this logic be moved into `PanZoomTool`? It would need a ref to the canvas to call `zoom`, etc.

Reply:
Yes, it's because this one tool would need a ref to the canvas, while all the others just get a callable, that I moved the logic into the canvases (and thus added duplication).
I think it's more consistent this way.